### PR TITLE
Added flask-minify from pypi

### PR DESCRIPTION
**Pull request by taluripooja**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by taluripooja** [comment]
@conda-forge/help-python  ready for review!

**Comment by taluripooja** [comment]
@conda-forge/staged-recipes ready for review

**Comment by taluripooja** [comment]
cc. @davidbrochart This PR is ready for review

File: LICENSE.txt [hunk 1]
```diff
@@ -1,27 +1,21 @@
-Copyright (c) 2015-2023, conda-forge
```
Comment (martinRenou) [inline LICENSE.txt:1]: This file should not be changed

File: recipes/Flask-Minify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.0
```
Comment (martinRenou) [inline recipes/Flask-Minify/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/Flask-Minify/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  run:
+    - flask
+    - htmlmin
+    - jsmin
+    - lesscpy
+    - python >=3.0
```
Comment (martinRenou) [inline recipes/Flask-Minify/meta.yaml:27]: As per https://github.com/mrf345/flask_minify/blob/master/setup.py#L5

```suggestion
    - python >=3.6
```

**Review by martinRenou — COMMENTED** [review]
Thanks! I have a couple of comments

**Comment by taluripooja** [comment]
@conda-forge/staged-recipes ready for review

**Comment by taluripooja** [comment]
cc. @martinRenou @davidbrochart This PR is ready for re-review

**Review by martinRenou — APPROVED** [review]
Thanks!

**Review by davidbrochart — APPROVED** [review]
LGTM, thanks!
I guess @mrf345 should comment to accept being a maintainer of this recipe.

**Comment by mrf345** [comment]
@taluripooja @davidbrochart I'm not familiar with this project, but happy to help in maintaining this

**Comment by davidbrochart** [comment]
@taluripooja We don't have rights to merge.

**Comment by taluripooja** [comment]
@davidbrochart Could you please inform me about the individual responsible for approving PR and excepted timeframe for its merging?

**Comment by davidbrochart** [comment]
You already pinged here, maybe you can ask on https://app.gitter.im/#/room/#conda-forge:matrix.org ?

**Comment by taluripooja** [comment]
I don't have access to [ https://app.gitter.im/#/room/#conda-forge:matrix.org]( https://app.gitter.im/#/room/#conda-forge:matrix.org)
@davidbrochart @martinRenou @SylvainCorlay @wolfv please assist me in merging this PR. Its quite important and urgent

**Comment by taluripooja** [comment]
Thank you everyone for your quick response.

**Pull request merged**